projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/utils.ts

```diff
@@ -2,3 +2,11 @@
  * Copyright 2021-2023 VMware, Inc.
  * SPDX-License-Identifier: Apache-2.0
  */
+
+export const TRANSFORM_JOB = 'Transform Job';

```

Comment:
The name of a variable should reflect its purpose or the kind of data it holds and should not be the actual data , right ?

So in this case a better more description and explicit name could b TRANSFORM_JOB_BUTTON_LABEL or something like that.

Reply:
Indeed. Changing them.